### PR TITLE
route table: properly display the default IPv6 route on Linux

### PR DESCRIPTION
**Pull request by schischi**
The routes table displays an IPv4 for the default IPv6 route:

   `osquery> SELECT destination,gateway FROM routes WHERE gateway <> "";`

   | destination | gateway               |
   | ----------- | --------------------- |
   | 0.0.0.0     | 172.17.208.1          |
   | 0.0.0.0     | fe80::5:73ff:fea0:514 |

New output:

   | destination | gateway               |
   | ----------- | --------------------- |
   | 0.0.0.0     | 172.17.208.1          |
   | ::          | fe80::5:73ff:fea0:514 |

Note: it works as expected on Darwin and FreeBSD.

### CODE REVIEW

File: osquery/tables/networking/linux/routes.cpp [hunk 1]
```diff
@@ -136,7 +139,11 @@ void genNetlinkRoutes(const struct nlmsghdr* netlink_msg, QueryData& results) {
   }
 
   if (!has_destination) {
-    r["destination"] = "0.0.0.0";
+    if (message->rtm_family == AF_INET) {
+      r["destination"] = kDefaultIpv4Route;
+    } else if (message->rtm_family == AF_INET6) {
+      r["destination"] = kDefaultIpv6Route;
+    }
```
Comment (guliashvili) [inline osquery/tables/networking/linux/routes.cpp]: else{
LOG(ERROR) maybe?

**Comment by guliashvili** [comment]
ok to test

**Comment by guliashvili** [comment]
ok to test

**Comment by guliashvili** [comment]
It will break backward compatibility. It's not blocking PR when it's needed though.

As I understand we need it because in case of the default value it was not possible to know if it's ipv6 or ipv4 right?

**Comment by schischi** [comment]
The default value for the route was 0.0.0.0 for both IPv4 and IPv6. '0.0.0.0' is not a valid IPv6 address ('::' is).

Without this patch, the output of the route table on Linux is wrong: it looks like an IPv4 destination has an IPv6 gateway.

**Comment by guliashvili** [comment]
LGTM.

**Pull request merged**